### DISCUSSION TITLE
Required to install Pandoc locally to run in Python?

### DISCUSSION THREAD

**Discussion by michellepace** [jgm/pandoc#9936] [Q&A] [answered]
Hi there. 

**Question 1:**
I wanted to use Pandoc in a Colab Notebook to convert my docx into markdown. From the Pandoc documentation am I correct that using Pandoc is not simply installing a python library, but in fact installing Pandoc locally? 

**Question 2**
If this is the case, does anyone perhaps know of a "good" python library I can use for docx into markdown? (and hopefully also markdown into html, although that is a bonus). 

Thanks again
Michelle

**Comment by denismaier** [comment, marked as answer]
ad 1: Yes, that's correct. I might be missing some context here,  but I don't see a connection to the Python ecosystem in any way. Pandoc is written in Haskell and can be used as a standalone executable and I think as a Haskell library. There exists a Python library that uses Pandoc (https://pypi.org/project/pandoc/), but as the documentation says, Pandoc needs to be available for this to work.

**Comment by michellepace** [comment]
Thank you very much @denismaier and apologies for not picking this up in the docs, I got too excited with the results when I tried pandoc in my web browser. 

Unfortunately I need to create markup from a word docx in a Colab notebook. I also need to then convert the markup returned back from the Anthropic Claude LLM model to html. I think I’m stuck with the python-docx library. Unsure what im going to use to convert my markdown to html.

**If anyone has a suggestion** I’d really appreciate it.

Thank-you you once again.
